buckwheat/tests/test_tokenizer.py

```diff
@@ -5,95 +5,149 @@
+                          "    Uses StatisticsHolder class for statistics information.\n    */",
+                          "// total number of subDirs"]],
                         ["Go", "test.go",
                          {"buff": 4, "file": 4, "fmt": 3, "fprintln": 3, "err": 3, "out": 2,
                           "string": 2, "name": 2, "languag": 2, "version": 1, "commit": 1,
                           "print": 1, "break": 1, "down": 1, "buffer": 1, "data": 1, "read": 1,
-                          "limit": 1}],
+                          "limit": 1},
+                         ["// Declaration of variables", "/* Breaking Down function\n   iterating over name and language\n*/"]],
                         ["C++", "test.cpp",
                          {"argc": 3, "argv": 3, "mini": 3, "maxi": 3, "std": 2, "atof": 2,
                           "acosd": 2, "main": 1, "zero": 1, "valmax": 1, "exhaust": 1, "test": 1,
-                          "pack": 1, "raw": 1}],
+                          "pack": 1, "raw": 1},
+                         ["// Initialize mini with zero", "/* Initialize maxi with maxval */"]],
                         ["Ruby", "test.rb",
                          {"out": 4, "tmp": 4, "string": 4, "get": 2, "block": 2, "new": 2,
-                          "stdout": 1, "stderr": 1}],
+                          "stdout": 1, "stderr": 1},
+                         ["# Declaration of get_stdout function", "=begin\nDeal with stderr\n=end"]],
                         ["TypeScript", "test.ts",
                          {"name": 10, "person": 5, "first": 4, "last": 4, "student": 2, "full": 2,
                           "middle": 2, "initi": 2, "greeter": 2, "user": 2, "constructor": 1,
-                          "document": 1, "body": 1, "text": 1, "content": 1}],
+                          "document": 1, "body": 1, "text": 1, "content": 1},
+                         ["/** Initialize user instance of Student class */"]],
                         ["TSX", "test.ts",
                          {"name": 10, "person": 5, "first": 4, "last": 4, "student": 2, "full": 2,
                           "middle": 2, "initi": 2, "greeter": 2, "user": 2, "constructor": 1,
-                          "document": 1, "body": 1, "text": 1, "content": 1}],
+                          "document": 1, "body": 1, "text": 1, "content": 1},
+                         ["/** Initialize user instance of Student class */"]],
                         ["PHP", "test.php",
                          {"set": 4, "key": 4, "content": 3, "file": 2, "quick": 2, "hash": 2,
                           "int": 2, "get": 1, "dirnam": 1, "load": 1, "from": 1, "string": 1,
                           "not": 1, "donot": 1, "use": 1, "zend": 1, "alloc": 1, "range": 1,
-                          "printf": 1, "exists": 1}],
+                          "printf": 1, "exists": 1},
+                         ["// read file to contents variable",
```

Comment:
let's create dataclass for this structure: language, filename, identifier counts, comments

Reply:
Great idea!